bundles/binding/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/LoginData.java

```diff
@@ -40,36 +39,27 @@
 	 *            The Json string as it has been returned myq website.
 	 */
 	@SuppressWarnings("unchecked")
-	public LoginData(String loginData) {
+	public LoginData(String loginData) throws IOException {
```

Comment:
I thought it would be cleaner to throw an exception if login fails rather than using a success boolean.